CgfConverter/Renderers/Wavefront/Wavefront.Material.cs

```diff
@@ -65,73 +65,77 @@ public void WriteMaterial(CryEngine cryEngine)
+                                textureFile = textureFile.Replace(".dds", ".png");
+                            else if (this.Args.TgaTextures)
+                                textureFile = textureFile.Replace(".dds", ".tga");
+                            else if (this.Args.TiffTextures)
+                                textureFile = textureFile.Replace(".dds", ".tif");
+
+                            switch (texture.Map)
+                            {
+                                case CryEngineCore.Material.Texture.MapTypeEnum.Diffuse:
+                                    file.WriteLine("map_Kd {0}", textureFile.ToString());
+                                    break;
+
+                                case CryEngineCore.Material.Texture.MapTypeEnum.Specular:
+                                    file.WriteLine("map_Ks {0}", textureFile.ToString());
+                                    file.WriteLine("map_Ns {0}", textureFile.ToString());
+                                    break;
+
+                                case CryEngineCore.Material.Texture.MapTypeEnum.Bumpmap:
+                                case CryEngineCore.Material.Texture.MapTypeEnum.Detail:
+                                    // <Texture Map="Detail" File="textures/unified_detail/metal/metal_scratches_a_detail.tif" />
+                                    file.WriteLine("map_bump {0}", textureFile.ToString());
+                                    break;
+
+                                case CryEngineCore.Material.Texture.MapTypeEnum.Heightmap:
+                                    // <Texture Map="Heightmap" File="objects/spaceships/ships/aegs/gladius/textures/aegs_switches_buttons_disp.tif"/>
+                                    file.WriteLine("disp {0}", textureFile.ToString());
+                                    break;
+
+                                case CryEngineCore.Material.Texture.MapTypeEnum.Decal:
+                                    // <Texture Map="Decal" File="objects/spaceships/ships/aegs/textures/interior/metal/aegs_int_metal_alum_bare_diff.tif"/>
+                                    file.WriteLine("decal {0}", textureFile.ToString());
+                                    break;
+
+                                case CryEngineCore.Material.Texture.MapTypeEnum.SubSurface:
+                                    // <Texture Map="SubSurface" File="objects/spaceships/ships/aegs/textures/interior/atlas/aegs_int_atlas_retaliator_spec.tif"/>
+                                    file.WriteLine("map_Ns {0}", textureFile.ToString());
+                                    break;
+
+                                case CryEngineCore.Material.Texture.MapTypeEnum.Custom:
+                                    // <Texture Map="Custom" File="objects/spaceships/ships/aegs/textures/interior/metal/aegs_int_metal_painted_red_ddna.tif"/>
+                                    // file.WriteLine("decal {0}", textureFile);
+                                    break;
```

Comment:
For cases that don't do anything, can you collapse them all together with the default case?

```
case CryEngineCore.Material.Texture.MapTypeEnum.Custom:
case CryEngineCore.Material.Texture.MapTypeEnum.BlendDetail:
case CryEngineCore.Material.Texture.MapTypeEnum.Environment:
default:
```

I think that works...